python_digest/__init__.py

```diff
@@ -58,7 +66,7 @@ def build_digest_challenge(timestamp, secret, realm, opaque, stale):
     '''
     nonce = calculate_nonce(timestamp, secret)
 
-    return b'Digest %s' % format_parts(realm=realm, qop='auth', nonce=nonce,
+    return 'Digest %s' % format_parts(realm=realm, qop='auth', nonce=nonce,
```

Comment:
It seems like this could break things if callers are expecting bytes. Can you explain how you know it's acceptable to change the return type of this function?

Reply:
```build_digest_challenge``` is used to assign the ```"WWW-Authenticate"``` response header.  Otherwise it's only used in tests.